slsim/Deflectors/richness2mass.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+
```

Comment:
Maybe write one line description for this module.

Reply:
added documentation